### PR TITLE
nfd-worker: don't log labels returned by sources by default

### PR DESCRIPTION
**Pull request by marquiz**
Reduce default log verbosity. Only print out labels if log verbosity is 1 or higher (e.g. '-v 1' command line). Also, dump the labels in a reproducible (sorted) format.

Produces logs like this with `-v 1`:
```
I0311 07:15:34.257115   23856 nfd-worker.go:523] labels discovered by feature sources:
I0311 07:15:34.257333   23856 nfd-worker.go:523]   cpu-cpuid.ADX: "true"
I0311 07:15:34.257346   23856 nfd-worker.go:523]   cpu-cpuid.AESNI: "true"
I0311 07:15:34.257367   23856 nfd-worker.go:523]   cpu-cpuid.AVX: "true"
I0311 07:15:34.257373   23856 nfd-worker.go:523]   cpu-cpuid.AVX2: "true"
I0311 07:15:34.257379   23856 nfd-worker.go:523]   cpu-cpuid.FMA3: "true"
I0311 07:15:34.257384   23856 nfd-worker.go:523]   cpu-cpuid.HYPERVISOR: "true"
I0311 07:15:34.257390   23856 nfd-worker.go:523]   cpu-cpuid.SSE4: "true"
I0311 07:15:34.257395   23856 nfd-worker.go:523]   cpu-cpuid.SSE42: "true"
```

### CODE REVIEW

File: pkg/nfd-worker/nfd-worker.go [hunk 1]
```diff
@@ -508,6 +508,7 @@ func createFeatureLabels(sources []source.FeatureSource, labelWhiteList regexp.R
 	labels = Labels{}
 
 	// Do feature discovery from all configured sources.
+	klog.Info("doing feature discovery...")
```
Comment (ArangoGutierrez) [inline pkg/nfd-worker/nfd-worker.go:511]: Ummm
starting feature discovery? or something better than `doing`? just a thought

Comment (marquiz) [reply pkg/nfd-worker/nfd-worker.go]: Mm, possibly not the best wording. Changed to *starting* as suggested and added an accompaniying *completed* log message after feature discovery has been finished

Comment (ArangoGutierrez) [reply pkg/nfd-worker/nfd-worker.go]: Looks good to me, thanks!

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**